pyproject.toml

```diff
@@ -10,5 +10,14 @@ authors = [
     {name = "Baptiste Mispelon", email = "hello@bmispelon.rocks"}
 ]
 license = {file = "LICENSE"}
-classifiers = ["License :: OSI Approved :: MIT License"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Framework :: Wagtail",
+    "Intended Audience :: Legal Industry",
+    "License :: OSI Approved :: MIT License",
```

Comment:
We should use either `License :: OSI Approved :: Sun Public License` or `License :: OSI Approved :: Nokia Open Source License` to show how modern we are (both only have one package with that license, so that's more visibility)

Reply:
This software is proudly EUropean, so I went with Nokia 🇫🇮